package.json

```diff
@@ -7,8 +7,14 @@
 	"author": {
 		"name": "Vadim Demedes",
 		"email": "vdemedes@gmail.com",
-		"url": "github.com/vadimdemedes"
+		"url": "https://github.com/vadimdemedes"
 	},
+	"contributors": [
+		{
+			"name": "Megrez Lu",
+			"email": "lujiajing1126@gmail.com"
+		}
+	],
```

Comment:
The `contributors` field is not used by npm or any tool and it's a chore to maintain. Your contribution will be reflected in the commit history and https://github.com/vadimdemedes/ink-select-input/graphs/contributors

Reply:
I see. Removed